draft-ietf-wimse-s2s-protocol.md

```diff
@@ -344,6 +344,8 @@ A WPT MUST contain the following:
      SHA-256 hash of the ASCII encoding of the associated token's value.
      (Note: this is less than ideal but seems we need something like this for extensibility.)
 
+To clarify: the `ath`, `tth` and `oth` claims are each mandatory if the respective token is included in the request.
+
```

Comment:
do we need to define what to do if there is more than one token that can fit in the oth claim?

Reply:
We could say something like, if there are multiple tokens they should be joined with colons as the separator, but I'm not sure this should be in scope: the whole `oth` things is basically an extension point.

Also, we would need to define an ordering between the tokens.

Reply:
Open issue #129